papas/path.py

```diff
@@ -1,10 +1,16 @@
 import math
 from scipy import constants
-from ROOT import TVector3
+from ROOT import TVector3, TLorentzVector
 from heppy.utils.deltar import deltaPhi
 from collections import OrderedDict
 import scipy.optimize as opti # need to compute impact parameters
 from numpy import sign
+import random as random
+from detectors.CMS import cms
+detector = cms
+
+epaisseur_beampipe = detector.elements['beampipe'].volume.outer.rad-detector.elements['beampipe'].volume.inner.rad
```

Comment:
the reason why we don't want any reference to the detector in this module is that users are going to want to use other detectors. This would lead to inconsistencies and bugs in the current code. Please have a look at the following comments and check that everything is consistent.